File: despatch.Domain/Models/Default/Response/ContactDataResponse.cs
using System;
using System.Collections.Generic;

namespace despatch.Domain.Models.Default.Response
{
    public class ContactDataResponse : BaseEntityResponse
    {
        public string DisplayName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MiddleName { get; set; }
        public string HonorificPrefix { get; set; }
        public string HonorificSuffix { get; set; }
        public string Nickname { get; set; }
        public string Note { get; set; }
        public string Email { get; set; }
        public DateTime? BirthdayDate { get; set; }

        public List<ContactDataAddressesResponse> Addresses { get; set; }
        public List<ContactDataOrganizationResponse> Organizations { get; set; }
        public List<ContactDataImageResponse> Images { get; set; }
        public List<ContactDataPhoneNumberResponse> PhoneNumbers { get; set; }
        public List<ContactDataUrlsResponse> Urls { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/contact-data-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ContactDataAddressesResponse } from './contact-data-addresses-response';
import { ContactDataImageResponse } from './contact-data-image-response';
import { ContactDataOrganizationResponse } from './contact-data-organization-response';
import { ContactDataPhoneNumberResponse } from './contact-data-phone-number-response';
import { ContactDataUrlsResponse } from './contact-data-urls-response';
export interface ContactDataResponse extends BaseEntityResponse{
  addresses?: Array<ContactDataAddressesResponse>;
  birthdayDate?: string;
  displayName?: string;
  email?: string;
  firstName?: string;
  honorificPrefix?: string;
  honorificSuffix?: string;
  images?: Array<ContactDataImageResponse>;
  lastName?: string;
  middleName?: string;
  nickname?: string;
  note?: string;
  organizations?: Array<ContactDataOrganizationResponse>;
  phoneNumbers?: Array<ContactDataPhoneNumberResponse>;
  urls?: Array<ContactDataUrlsResponse>;
}
